### DISCUSSION TITLE
[Enhancement] Mob Sunscreen behavior but performant

### DISCUSSION THREAD

**Discussion by NoMoreUsernames999** [ACGaming/UniversalTweaks#486] [Tweak Suggestions]
So I recently discovered that the mod Mob Sunscreen is terrible for performance, as it checks every mob every tick. It also doesn't properly implement its feature, since mobs stepping into half blocks like PVJ leaf piles will still catch fire. 

So my suggestion is to incorporate a proper mob sunscreen function into UT. Here is the conversation where someone suggested a mixin for just disabling zombies:
![Screenshot_20240608-104803](https://github.com/ACGaming/UniversalTweaks/assets/67126196/b383abb0-7f29-4675-b8d5-219f2b3fd2d6)

And here is the class from a modern alternative to Mob Sunscreen which disables burning for everything (not sure how applicable it is to 1.12 code):
https://github.com/muhammedesadcomert/Protect-Mobs-From-DayLight/blob/multiloader-1.19%2B/common/src/main/java/com/muhammedesadcomert/protectmobsfromdaylight/common/mixin/MobMixin.java

However you plan to implement this, If you plan to do so, all I really want is to just disable mob burning completely without murdering my TPS. If there is a way to selectively do it that's still performant, I guess that would be useful for other people too?

**Comment by ACGaming** [comment]
Implemented in eed1eab73b06ae366240b8057c03bde0c15dccd5.
